sequence_processing_pipeline/GenPrepFileJob.py

```diff
@@ -29,17 +30,25 @@ def __init__(self, run_dir, convert_job_path, qc_job_path, output_path,
                  join(self.output_path, self.run_id, 'Reports'))
 
         for project in project_list:
-            src1 = join(qc_job_path, project, 'filtered_sequences')
-            src2 = join(qc_job_path, project, 'fastp_reports_dir', 'json')
+            filtered_seq_dir = join(qc_job_path, project, 'filtered_sequences')
```

Comment:
Thank you for cleaning, here it might useful to use `partial`

Reply:
Good point!